src/core/components/profile/registercard.component.tsx

```diff
@@ -95,12 +97,12 @@ const RegisterCard: React.FC<RegisterCardProps> = props => {
           เพื่อมาเป็นส่วนหนึ่งในความสนุกที่ไม่สามารถหาจากที่อื่นได้อีก!
         </Typography>
         {props.profileStatus === "start" && (
-          <Button variant="contained" disableElevation className={`${classes.button} ${classes.solid}`}>
+          <Button variant="contained" disableElevation className={`${classes.button} ${classes.solid}`} onClick={handleClick}>
```

Comment:
can you use `Link` component from `react-router-dom`

Reply:
ตอนแรกสุดใช้ Link แล้วเป็นแบบนี้ ก็เลยไม่ใช้ 
เข้าใจว่าเส้นมันคือเส้นจากลิงก์ แต่พยายามใช้ text-decoration: "none" หรือ underline ก็ไม่ออกอะ

https://user-images.githubusercontent.com/57129145/141083412-0e2650be-85e5-44df-b96b-0cb591203363.mp4

้

Reply:
อ้อ ใส่ className ชื่อ `no-underline` ด้วยๆ

Reply:
มันยึกยักเหมือนเดิมแค่ไม่เห็นเส้นแล้ว...

Reply:
That's weird

Reply:
มันก็มีหลายจุดเหมือนกันที่ใช้ `no-underline` ลองไปก๊อปๆมาก็ได้